recipes/opensubdiv/all/conanfile.py

```diff
@@ -86,7 +86,14 @@ def requirements(self):
             if Version(self.version) < "3.6.0":
                 self.requires("onetbb/2020.3.3", transitive_headers=True)
             else:
-                self.requires("onetbb/2021.10.0", transitive_headers=True)
+                self.requires("onetbb/2021.12.0", transitive_headers=True)
+        if self.options.get_safe("with_opengl"):
+            self.requires("opengl/system")
+            self.requires("glfw/3.4")
+        if self.options.get_safe("with_metal"):
+            self.requires("metal-cpp/14.2")
+        if self.options.get_safe("with_dx"):
```

Comment:
On the other hand @EstebanDugueperoux2,
Why is this requirement needed?
I've tested locally on a Windows machine and see no difference on build logs when compiling with `with_dx=True` on both `master` and this PR branch.

Also, if you enable `examples` (line 140), and `with_dx`, you will have a compilation error:

```
d3d11ShaderCache.obj : error LNK2001: unresolved external symbol D3DCompile [C:\Users\perseo\.conan2\p\b\opensec2a33492099e\b\build\examples\dxViewer\dxViewer 
.vcxproj]
```

Do you have any idea?
Thank you!

Reply:
Hi @perseoGI ,
thanks for you test, I have no windows machine on my side.
Indeed DX11SDK seems used only by examples and as this last are disabled by default, I'll rollback my changes about with_dx option.

Reply:
Greetings @EstebanDugueperoux2,
Thank you for your rollback.
The easiest way to get a PR reviewed and merged is having all the changes in the PR under control and with a justification & description in the details box.
If you consider those DX were needed, it may be easier to open another PR after this one gets merged with a more descriptive title and information.

Thank you in advance!
I'm approving again the PR 🐸